transactions/setup_account_from_nft_reference.cdc

```diff
@@ -0,0 +1,28 @@
+import NonFungibleToken from "../contracts/NonFungibleToken.cdc"
+import MetadataViews from "../contracts/MetadataViews.cdc"
+import ExampleNFT from "../contracts/ExampleNFT.cdc"
+
+transaction(address: Address, id: UInt64) {
```

Comment:
```suggestion
transaction(address: Address, publicPath: PublicPath, id: UInt64) {
```
Would it make sense to make the collection public path an argument to this also, so that when the other cadence feature is complete, we can do everything without having to import the `ExampleNFT` contract?